doc/arch/adr-001-stop-app-to-stop-sending.md

```diff
@@ -0,0 +1,15 @@
+# ARD 001: Stop app to stop sending
```

Comment:
file name uses 'adr', this line uses 'ard', which one is correct?

Reply:
adr ;) architecture decision register